src/studio/src/designer/frontend/dashboard/services/designerApi.ts

```diff
@@ -12,6 +13,13 @@ export const designerApi = createApi({
   tagTypes: Object.values(TagTypes),
   baseQuery: fetchBaseQuery({
     baseUrl: `${window.location.origin}/designer/api/v1`,
+    prepareHeaders: (headers) => {
+      const xsrfToken = getCookie('XSRF-TOKEN');
+
+      headers.set('X-XSRF-TOKEN', xsrfToken);
```

Comment:
Does this work as expected? 😍  I had trouble trying this in app-frontend as fetch appears to be setting the headers with lowercase and that caused trouble when backend was validating header

Reply:
I did not encounter any issues with this approach at least, and it works as expected.

Headers are (or at least should be) case insensitive according to the spec https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html (section 4.2), so if we have any code that relies on casing in headers we should investigate and fix that.